apps/bors_frontend/web/controllers/webhook_controller.ex

```diff
@@ -79,11 +80,30 @@ defmodule BorsNG.WebhookController do
   @doc """
   This action is reached via `/webhook/:provider`
   """
-  def webhook(conn, %{"provider" => "github"}) do
-    event = hd(get_req_header(conn, "x-github-event"))
-    do_webhook conn, "github", event
-    conn
-    |> send_resp(200, "")
+  def webhook(conn, %{"provider" => provider}) do
+    case provider do
+      "github" ->
+        event = hd(get_req_header(conn, "x-github-event"))
+        do_webhook conn, "github", event
+
+      "bitbucket" ->
+        event = hd(get_req_header(conn, "x-event-key"))
+        Logger.info "Event: #{event}"
+        do_bitbucket_webhook conn, event
+
+        _ ->
+          conn |> send_resp(404, "")
+    end
+
+    conn |> send_resp(200, "")
+  end
+
+  def do_bitbucket_webhook(conn, "pullrequest:comment") do
+    comment = conn.body_params["comment"]
+    Logger.info "comment: #{comment}"
+     if String.contains?(comment, "bors") do
```

Comment:
@notriddle any reason why you don't do something like this before starting a command?